src/d3/real-time-mouse-overlay.js

```diff
@@ -54,7 +54,7 @@ function realTimeMouseOverlay() {
         .attr('height', height - (bubbleWidth / 2))
         .attr('width', width - bubbleWidth);
 
-      const mouseTargetEnter = mouseTarget.enter()
```

Comment:
is it necessary to remove the named variable too? Or does the else block and resetting the event handlers already fix the issue?

Reply:
Yes, it is! Using that variable does not allow the data to be updated correctly which does not allow scrubbing regardless of whether the mouse is enabled or not. The else statement handles a separate issue which distinguishes when the graph is paused vs. when it is being played.